pkg/plugin/istio/install.go

```diff
@@ -486,8 +502,8 @@ func (p *IstioPlugin) iopFiles(cluster string) ([]string, error) {
 	iopFiles = append(iopFiles, p.args.IopFiles...)
 	if !p.IsFlat() {
 		mesh := clusterNetwork{
-			Network:     networkName(cluster), // TODO: support custom network, e.g. primary and remote1 in network1, remote2 in network2
-			MeshID:      "mesh1",              // TODO: make this configurable
+			Network:     p.clusterNetwork(cluster),
```

Comment:
why remove the TODO，IMO this is not resoloved by this patch

Reply:
mistake

Reply:
you can label on cluster before run `kurator install istio` to achieve this

Reply:
I mean do you support 

`e.g. primary and remote1 in network1, remote2 in network2`

From my reading, it seems even `primary and remote1 in network1`, we still added east-west gateway

Reply:
yes, it should be improved, let's track that with an issue.